lighthouse-core/test/gather/driver-test.js

```diff
@@ -952,12 +901,7 @@ describe('Multi-target management', () => {
     });
     await flushAllTimersAndMicrotasks();
 
-
-    const sendMessageArgs = connectionStub.sendCommand
-      .findInvocation('Target.sendMessageToTarget');
-    expect(sendMessageArgs).toEqual({
-      message: JSON.stringify({id: 1, method: 'Runtime.runIfWaitingForDebugger'}),
-      sessionId: 'SW1',
-    });
+    expect(connectionStub.sendCommand)
```

Comment:
feels like we might actually want to use `toHaveBeenNthCalledWith` for a stronger guarantee that nothing else was called if we're using it in the previous test case

Reply:
It's being used in the previous case out of necessity - AFAICT `toHaveBeenCalledWith` (also used by `findInvocation`) only verifies against the very last call.

I could instead use `findInvocation` here, it's just odd b/c it returns `undefined` and all the other usages of it verifies the return result in some way. Also, technically right now we can't  use `findInvocation` for commands issued w/o any args. see:

![image](https://user-images.githubusercontent.com/4071474/66505751-ce568180-ea80-11e9-9db4-95fdade9c876.png)

![image](https://user-images.githubusercontent.com/4071474/66505799-e6c69c00-ea80-11e9-806b-247abd362cff.png)



^that's one annoying reason to avoid `findInvocation` for this one use case, I guess.

Reply:
I don't know why `findInvocation` errors here. It's confusing - but `toHaveBeenNthCalledWith` is easy to reason about, so probably should just stick with that.

![image](https://user-images.githubusercontent.com/4071474/66506103-884ded80-ea81-11e9-9606-3c740b2cede8.png)


![image](https://user-images.githubusercontent.com/4071474/66506092-8421d000-ea81-11e9-9d56-41abffd21634.png)

Reply:
OK I'll rephrase my comment :)

Pre-existing, but it'd be nice to add the assertion here that we are not listening to network traffic of non-iframe subtargets so how about we use the same clever `toHaveBeenNthCalledWith` solution from before? :D